mlflow/tracing/utils/__init__.py

```diff
@@ -13,6 +13,7 @@
 from opentelemetry import trace as trace_api
 from packaging.version import Version
 
+import mlflow
```

Comment:
Should we explicitly import only the `MlflowClient` interface and `get_current_active_span` fluent API to avoid a broad import to this utility module?